python/ray/train/callbacks/logging.py

```diff
@@ -86,8 +86,8 @@ def _create_log_path(self, logdir_path: Path, filename: Path) -> Path:
             raise ValueError("filename cannot be None or empty.")
         return logdir_path.joinpath(Path(filename))
 
-    def start_training(self, logdir: str):
-        super().start_training(logdir)
+    def start_training(self, logdir: str, config: Dict, **info):
+        super().start_training(logdir, config, **info)
```

Comment:
Do these explicitly need `config` or would that be captured with `**info`? 

Having the same args in every `start_training` signature will surely work, but it seems like the original intention here was to keep the arg list of each implementation specific to what's being used. Is there a patten we can follow to have both extensibility and simplicity?

Reply:
AFAIK `**info` is there only for forwards-compatibility - all arguments should be explicitly defined regardless. At least that's how it is in Tune

Reply:
Right, isn't the point of forwards compatibility here that when you add a new argument to `TrainingCallback.start_training`, you don't need to add that argument to every existing overridden implementation?

Reply:
That is a good point. I'll leave only what's being used.